include/seqan3/alignment/scoring/gap_scheme.hpp

```diff
@@ -264,9 +264,9 @@ class gap_scheme
     //!\endcond
 
 private:
-    //!\brief The score per gap character.
-    score_t gap = 0;
-    //!\brief The score per sequence of gaps.
+    //!\brief The score per gap character. Defaults to -1.
+    score_t gap = -1;
```

Comment:
Is this safe, if the user defines score_t as unsigned?
Or do you make sure that unsigned types are prohibited?

Reply:
Well if the user makes them unsigned, he needs to do it explicitly. So the user should know what he is doing